pkg/global/constant_rate_trace_sampler.go

```diff
@@ -0,0 +1,86 @@
+package global
+
+import (
+	"sync"
+	"time"
+
+	"github.com/buildbarn/bb-storage/pkg/clock"
+
+	"go.opencensus.io/trace"
+)
+
+// ConstantRateTraceSampler uses a token bucket algorithm to sample traces. This is useful as an alternative
```

Comment:
Please wrap comments at <80 columns.

Reply:
Will do.

I'll make the manual edit for now, but would love to know if there is any automated tooling for Go that can do this consistently.

Reply:
and if so, maybe incorporate into the CI flow?